mgtool/exec.go

```diff
@@ -37,3 +39,14 @@ func run(cmd, dir string, stdout, stderr io.Writer, args ...string) error {
 
 	return c.Run()
 }
+
+func Command(path string, args ...string) *exec.Cmd {
+	cmd := exec.Command(path)
+	cmd.Args = append(cmd.Args, args...)
+	cmd.Dir = mgpath.FromGitRoot(".")
+	cmd.Env = os.Environ()
+	cmd.Stderr = os.Stderr
+	cmd.Stdout = os.Stdout
+	cmd.Stdin = os.Stdin
```

Comment:
IMO this might not be what we want

Reply:
imagine multiple commands running in parallel, all trying to read from the parent process stdin